rmw/include/rmw/types.h

```diff
@@ -166,6 +170,9 @@ typedef struct RMW_PUBLIC_TYPE rmw_subscription_t
 
   /// Indicates whether this subscription can loan messages
   bool can_loan_messages;
+
+  /// Indicates whether this subscription can support content filtered topic
+  bool is_cft_supported;
```

Comment:
Just a comment. this kind of feature `flag` would increase in the future. i would create bitmap field and each bits match against each feature. 

e.g)

```
#define LOAN_MESSAGE_BIT_SHIFT 0x0
#define CTF_BIT_SHIFT 0x1

#define LOAN_MESSAGE (1<< LOAN_MESSAGE_BIT_SHIFT)
#define CTF (1<< CTF_BIT_SHIFT)

#define IS_CTF_SUPPORTED (feature_flags && CTF)
```

Reply:
Actually, I don't know whether this usage is good or not at here. (Pros: less space, Cons: the `feature_flags` is not easy to read in gdb).
I think we can use another PR to fix this later. (NOTE: I don't want to update other unrelated source code In this PR.)